sql/001_create_volunteers.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE `volunteers` (
```

Comment:
Good start. Please add the statements for the other tables as well to other files (using the same file format: 00X_description.sql).